tests/test_target.py

```diff
@@ -23,14 +23,13 @@ def test_render_target():
     target = cluster._full_target(cluster_obj, components, catalog)
     facts = cluster_obj['facts']
     assert target != ""
-    all_classes = ([f"defaults.{cn}" for cn in components] +
-                   ['global.common',
-                       f"global.{facts['distribution']}",
-                       f"global.{facts['cloud']}",
-                       f"{cluster_obj['tenant']}.{cluster_obj['id']}",
-                    ])
-    assert len(target['classes']) == len(
-        all_classes), "rendered target includes different amount of classes"
+    all_classes = [f"defaults.{cn}" for cn in components] + \
+        ['global.common',
+         f"global.{facts['distribution']}",
+         f"global.{facts['cloud']}",
+         f"{cluster_obj['tenant']}.{cluster_obj['id']}"]
```

Comment:
I usually try to use the power of braces.

* They make the use of backslashes unneeded
* The code structure remains as we're used to function bodies in Java, JavaScript, CSS, etc.

```suggestion
    all_classes = [f"defaults.{cn}" for cn in components] + [
        'global.common',
        f"global.{facts['distribution']}",
        f"global.{facts['cloud']}",
        f"{cluster_obj['tenant']}.{cluster_obj['id']}",
    ]
```
Try to understand if you like this more. (I, personally, do prefer it this way.)